File: 2020/1/1.py
f = open("input")
lines = f.readlines()
f.close()


for i in range(0,len(lines)-1):
    for j in range(i+1,len(lines)-1) :
        for k in range(j+1,len(lines)-1) :
            if int(lines[i])+int(lines[j])+int(lines[k]) == 2020 :
                print int(lines[i])*int(lines[j])*int(lines[k])

File: 2021/1/1.go
package main

import (
    "bufio"
    "fmt"
    "log"
    "os"
	"strconv"
)

func main() {
    file, err := os.Open("input")
    if err != nil {
        log.Fatal(err)
    }
    defer file.Close()

    scanner := bufio.NewScanner(file)
	scanner.Scan()
    val, err := strconv.Atoi(scanner.Text())
	count := 0
	var newVal int
    for scanner.Scan() {
        newVal, err = strconv.Atoi(scanner.Text())
		if newVal > val {
			count++
		}
		val = newVal
    }

    if err := scanner.Err(); err != nil {
        log.Fatal(err)
    }

	fmt.Println(count)
}